### DISCUSSION TITLE
Align Images by Middle, Bottom or Top

### DISCUSSION THREAD

**Discussion by RoyiAvital** [quarto-dev/quarto-cli#3508] [Q&A] [answered]
Hi,

Assuming I have a set of images defined by:

```
![](https://i.imgur.com/vU9I0qc.png){width="350"}
![](https://i.imgur.com/2XYIwod.png){width="350"}
![](https://i.imgur.com/7Ln7cLV.png){width="350"}
```

By default they are aligned by their bottom (Namely if they all get in horizontally, their bottom is aligned).
Is there a way to control it? To have the elements aligned by their top or middle?

**Comment by mcanouil** [comment, marked as answer]
You mean like the following?

```md
::: {layout-ncol=3 layout-valign="center"}
![](https://i.imgur.com/vU9I0qc.png)

![](https://i.imgur.com/2XYIwod.png)

![](https://i.imgur.com/7Ln7cLV.png)
:::
```

Source: <https://quarto.org/docs/authoring/figures.html#vertical-alignment>.

**Reply by RoyiAvital** [reply]
Will that also resize the image to match the 3 columns?

**Reply by RoyiAvital** [reply]
By the way, I get:

![image](https://user-images.githubusercontent.com/7347975/204656683-98ae209c-1e20-4cdd-87fc-fcb6043fe822.png)

Maybe another name?

**Reply by RoyiAvital** [reply]
OK, The key should be `"center"` instead of `"middle"`:

```
::: {layout-ncol=3 layout-valign="center"}
![](https://i.imgur.com/vU9I0qc.png)

![](https://i.imgur.com/2XYIwod.png)

![](https://i.imgur.com/7Ln7cLV.png)
:::
```

The image is contained within the column.

@mcanouil , Could you edit your answer so I can mark it?

Thank You.

**Reply by mcanouil** [reply]
"center" in html is usually for horizontal alignment while "middle" is for vertical alignment.
This is not the case in Quarto. In VSCode, intellisense shows the proper options, but I only wrote it on my mobile from what I recall and the documentation.

**Reply by RoyiAvital** [reply]
@mcanouil , I agree. It is also the notation used on PowerPoint and such...

Anyhow, you solved my problem.
